docs/client/python/pyspark.md

```diff
@@ -106,6 +106,32 @@ jdbcDF.write \
     .save()
 ```
 
+### Using as JDBC Datasource table with DDL
```

Comment:
DDL => SQL ?

Reply:
'DDL' here means usage of "create table using jdbc" command. Any better idea for this title?

Reply:
the content contains other SQL-style examples as well

Reply:
ok, well known.